salvo/install_deps_ci.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
```

Comment:
Should we add `set -eo pipefail`?

Reply:
Yes, we should add `set -eo pipefail` in all CI scripts. Updated, thanks.